go/porcelain/deploy.go

```diff
@@ -404,21 +387,27 @@ func (n *Netlify) uploadFile(ctx context.Context, d *models.Deploy, f *FileBundl
 
 		switch t {
 		case fileUpload:
-			params := operations.NewUploadDeployFileParams().WithDeployID(d.ID).WithPath(f.Name).WithFileBody(f)
-			if timeout != 0 {
-				params.SetTimeout(timeout)
+			body, operationError := os.Open(f.Path)
```

Comment:
doesn't this mask the operationError? https://play.golang.org/p/R2gPzwewg3u

Reply:
Yes, good catch.